db/migrate/20180711120130_add_missing_ems_id_to_switch.rb

```diff
@@ -0,0 +1,42 @@
+class AddMissingEmsIdToSwitch < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+
+    belongs_to :ext_management_system, :foreign_key => "ems_id"
+
+    has_many :host_switches
```

Comment:
Any relationship needs to fully qualify the stubs.. e.g.  `:class_name => "AddMissingEmsIdToSwitch::HostSwitch"`

Reply:
this is used only in spec, the migration is using raw SQL